test/e2e/kubectl.go

```diff
@@ -32,22 +32,18 @@ import (
 )
 
 const (
-	nautilusImage       = "kubernetes/update-demo:nautilus"
-	kittenImage         = "kubernetes/update-demo:kitten"
-	updateDemoSelector  = "name=update-demo"
-	updateDemoContainer = "update-demo"
-	kubectlProxyPort    = 8011
+	nautilusImage            = "kubernetes/update-demo:nautilus"
+	kittenImage              = "kubernetes/update-demo:kitten"
+	updateDemoSelector       = "name=update-demo"
+	updateDemoContainer      = "update-demo"
+	frontendSelector         = "name=frontend"
+	redisMasterSelector      = "name=redis-master"
+	redisSlaveSelector       = "name=redis-slave"
+	kubectlProxyPort         = 8011
+	guestbookResponseTimeout = 2 * time.Minute
```

Comment:
To add a bit of explanation, the longer timeout helps reduce flakiness from slow, serialized docker image pulls (see #4566).